CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.5.0 (2020-09-01)
+
+- Added support for `--chromeDriver` flag during `skyux e2e`, allowing you to bypass the running of `chromedriver-manager update`. [#323](https://github.com/blackbaud/skyux-sdk-builder/pull/323)
```

Comment:
Can we tighten this up to: "Added the `--chromeDriver` flag to bypass `chromedriver-manager update` during `skyux e2e`."?

Also, is it correct that `chromedriver-manager update` is a code snippet? Looking at the code, it looks like maybe this code snippet should be `chromeDriverManager.update`. Or maybe `webdriverManager.update`. Or should refer to it generically, like "...to bypass the ChromeDriver Manager update during `skyux e2e`"?

Reply:
I like the last, generic suggestion. Would you mind pushing that and the date change to this branch?

Reply:
Will do.

Reply:
Thanks John!